doc/modules/cassandra/pages/new/configuration.adoc

```diff
@@ -0,0 +1,198 @@
+|batch_size_warn_threshold_in_kb |batch_size_warn_threshold |KiB
+|batch_size_fail_threshold_in_kb |batch_size_fail_threshold |KiB
+|compaction_throughput_mb_per_sec |compaction_throughput |MiB/s
+|compaction_large_partition_warning_threshold_mb |compaction_large_partition_warning_threshold |MiB
+|min_free_space_per_drive_in_mb |min_free_space_per_drive |MiB
+|stream_throughput_outbound_megabits_per_sec |stream_throughput_outbound |MiB/s
+|inter_dc_stream_throughput_outbound_megabits_per_sec |inter_dc_stream_throughput_outbound |MiB/s
+|commitlog_total_space_in_mb |commitlog_total_space |MiB
+|commitlog_sync_group_window_in_ms |commitlog_sync_group_window |ms
+|commitlog_sync_period_in_ms |commitlog_sync_period |ms
+|commitlog_segment_size_in_mb |commitlog_segment_size |MiB
+|periodic_commitlog_sync_lag_block_in_ms |periodic_commitlog_sync_lag_block |ms
+|max_mutation_size_in_kb |max_mutation_size |KiB
+|cdc_total_space_in_mb |cdc_total_space |MiB
+|cdc_free_space_check_interval_ms |cdc_free_space_check_interval |ms
+|dynamic_snitch_update_interval_in_ms |dynamic_snitch_update_interval |ms
+|dynamic_snitch_reset_interval_in_ms |dynamic_snitch_reset_interval |ms
+|hinted_handoff_throttle_in_kb |hinted_handoff_throttle |KiB
+|batchlog_replay_throttle_in_kb |batchlog_replay_throttle |KiB
+|hints_flush_period_in_ms |hints_flush_period |ms
+|max_hints_file_size_in_mb |max_hints_file_size |MiB
+|trickle_fsync_interval_in_kb |trickle_fsync_interval |KiB
+|sstable_preemptive_open_interval_in_mb |sstable_preemptive_open_interval |MiB
+|key_cache_size_in_mb |key_cache_size |MiB
+|row_cache_size_in_mb |row_cache_size |MiB
+|counter_cache_size_in_mb |counter_cache_size |MiB
+|networking_cache_size_in_mb |networking_cache_size |MiB
+|file_cache_size_in_mb |file_cache_size |MiB
+|index_summary_capacity_in_mb |index_summary_capacity |MiB
+|index_summary_resize_interval_in_minutes |index_summary_resize_interval |m
+|gc_log_threshold_in_ms |gc_log_threshold |ms
+|gc_warn_threshold_in_ms |gc_warn_threshold |ms
+|tracetype_query_ttl |trace_type_query_ttl |s
+|tracetype_repair_ttl |trace_type_repair_ttl |s
+|prepared_statements_cache_size_mb |prepared_statements_cache_size |MiB
+|enable_user_defined_functions |user_defined_functions_enabled |-
+|enable_scripted_user_defined_functions |scripted_user_defined_functions_enabled |-
+|enable_materialized_views |materialized_views_enabled |-
+|enable_transient_replication |transient_replication_enabled |-
+|enable_sasi_indexes |sasi_indexes_enabled |-
+|enable_drop_compact_storage |drop_compact_storage_enabled |-
+|enable_user_defined_functions_threads |user_defined_functions_threads_enabled |-
+|enable_legacy_ssl_storage_port |legacy_ssl_storage_port_enabled |-
+|===
+
+Another TO DO is to add new JMX methods supporting the new format. This might never happen though if virtual tables provide
+support to change our configuration sooner than later. Discussions and preliminary work are in the making.
```

Comment:
Alternate wording:

> Preliminary work is underway to add JMX methods that support the new value format. However, we may abandon this if virtual tables support for configuration changes in the near future.

Reply:
We don't have tickets we actively work on to do it, so not sure that preliminary work is better than TO DO. The rest I will incorporate

Reply:
That is a good point now though when a release is approaching. I will double-check with Benjamin as he had some ticket, I am not sure whether he will have the time to finish it before this release.